conftest.py

```diff
@@ -2,9 +2,19 @@
 import sys
 import uuid
 
+import coiled
 import pytest
-from coiled._beta import ClusterBeta as Cluster
 from dask.distributed import Client
+from packaging.version import Version
+
+# Use non-declarative clusters on Python 3.7 with `coiled <= 0.0.73`
+# See https://github.com/coiled/coiled-runtime/pull/54
+if tuple(sys.version_info[:2]) < (3, 8) and Version(coiled.__version__) <= Version(
+    "0.0.73"
+):
+    from coiled import Cluster
+else:
+    from coiled._beta import ClusterBeta as Cluster
```

Comment:
cc @dchudz for thoughts (though not urgent, you should feel free to not get back to me today)

Reply:
Seems fine as a temporary hack.

We're literally deploying the fix for 3.7 tonight, but I understand the desire to be decoupled from us, so this is cool.